go/ql/lib/semmle/go/dataflow/internal/DataFlowNodes.qll

```diff
@@ -718,7 +759,14 @@ module Public {
     CallNode c;
     int i;
 
-    ArgumentNode() { this = getArgument(c, i) }
+    ArgumentNode() {
+      exists(CallExpr call | call = c.getCall() |
+        this = getArgument(c, i) and not this.asExpr() = call.getImplicitVarargsArgument(_)
```

Comment:
Is `and not this.asExpr() = call.getImplicitVarargsArgument(_)` ever false? The implicit arg is wrapped into a slice, and therefore shouldn't be a direct argument per `getArgument`?

Reply:
Yes, that piece of code can be false. There is some ambiguity about whether arguments which are wrapped into a slice and passed to a variadic parameter should be called arguments or not. Currently they are returned by `getArgument` on `CallExpr` and `getArgument` on `CallNode`, but they aren't given argument nodes. I admit this is confusing, and I'm happy to change it for the sake of clarity, though it's unclear exactly what to change it to.